bindings/c/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "slatedb-c"
+version = "0.1.0"
```

Comment:
Is it possible to peg this version to be the same as the root Cargo.toml (ideally without hardcoding it)? I am thinking we should keep the version bindings in lock-step with the Rust version so it's easier to understand what features are available (and what code is being used).